### ISSUE TITLE
App Design

### ISSUE THREAD

**Issue by umutphp** [ayse8888/random-quotes#2]
Can you please change the look of the application as close as the screenshot below?

![design](https://user-images.githubusercontent.com/3245166/144137902-8389276b-a0bb-4337-870e-03d7ff857bb0.png)

PS: It is just for checking your UI skills more :)

**Comment by ayse8888** [comment]
I changed the design of the app accordingly :)
It looks as below on my phone.
I used https://imagecolorpicker.com/ for the colors.
I sent my new APK key via e-mail.

![quotes](https://user-images.githubusercontent.com/65858467/144306552-2888102d-19b1-401d-af56-8614379832b9.jpeg)

**Comment by umutphp** [comment]
@ayse8888 many thanks for the implementation. I have two queries about the app.

### Bug 1
The dark blocks are not 100% wide on my phone. If the quote is shorter, the white margins are getting bigger.
![Screenshot_20211202-000118_quotes-app](https://user-images.githubusercontent.com/3245166/144314008-b6474272-0799-4cef-b491-b1c70b5c80a6.jpg)


### Bug 2
The tag boxes under the quote should be dynamic as the quotes. The app always display two boxes.

**Comment by ayse8888** [comment]
Hi Mr Umut :)

1. for the first bug, I had given padding : 0 for the biggest container and margin : 0 for the inside element before. But apparently it did not work and I could not figure out why. Thus I gave a width to my card element to fix this bug. Now margins are disappeared.
2.  for the second bug. I made the tag names dynamically. 

Thank you so much and I shared my new APK file via gmail again.
